### DISCUSSION TITLE
360 panoramas

### DISCUSSION THREAD

**Discussion by romanIlyushin-bc** [visgl/deck.gl#5239] [Ideas]
Is it possible to create Google Street View-like visualization with equirectangular panorama images based on Bitmap Layer and FirstPersonView?

**Comment by Pessimistress** [comment]
BitmapLayer only renders a flat surface. You may look into the SimpleMeshLayer and create a textured sphere/cylinder.

**Comment by chrisgervang** [comment]
We've just added a 360 video example https://github.com/visgl/deck.gl/pull/5414. It works with images and many other formats too, just replace the texture with an image link.
